web/src/styles/messages.css

```
@@ -272,8 +272,42 @@ html[data-theme] .interview-bar-actions .btn-primary:hover { background: var(--t
-.composer-input { flex: 1; border: none; background: transparent; font-family: var(--font-sans); font-size: 14px; color: var(--text); outline: none; resize: none; min-height: 28px; max-height: 120px; line-height: 1.5; padding: 4px 0; align-self: center; }
+/* Mirror + textarea share a relative container so their text lays out on
+   the same pixel grid. Font metrics (family, size, color) are set by the
+   active theme (nex.css etc.) so a single place controls both surfaces. */
+.composer-field { position: relative; flex: 1; min-height: 28px; align-self: center; }
+.composer-mirror,
+.composer-input {
+  line-height: 1.5;
+  padding: 4px 0;
+  min-height: 28px;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)